app/parse.py

```diff
@@ -8,8 +15,60 @@
     tags: list[str]
 
 
+QUOTE_FIELDS = ["text", "author", "tags"]
+
+HOME_PAGE = "https://quotes.toscrape.com/"
+
+
+def parse_single_quote(quote_element: Tag) -> [Quote]:
+    return Quote(
+        text=quote_element.select_one(".text").text,
+        author=quote_element.select_one(".author").text,
+        tags=[tag.text for tag in quote_element.select(".tag")],
+    )
+
+
+def get_next_page(page_soup:  BeautifulSoup) -> str | None:
```

Comment:
The return type annotation `str | None` is not valid in Python versions before 3.10. If you are using an earlier version, consider using `Optional[str]` from the `typing` module instead.